AndroidSDKCore/src/main/java/com/leanplum/internal/Util.java

```diff
@@ -557,6 +557,10 @@ static HttpURLConnection createHttpUrlConnection(
         getApplicationName(context) + "/" + getVersionName() + "/" + RequestOld.appId() + "/" +
             Constants.CLIENT + "/" + Constants.LEANPLUM_VERSION + "/" + getSystemName() + "/" +
             getSystemVersion() + "/" + Constants.LEANPLUM_PACKAGE_IDENTIFIER);
+    if (Leanplum.featureFlagManager().isFeatureFlagEnabled(Leanplum.featureFlagManager().FEATURE_FLAG_REQUEST_GZIP)) {
+      urlConnection.setRequestProperty("Accept-Encoding", "gzip");
+      Leanplum.countAggregator().incrementCount("operation_with_request_gzip");
+    }
```

Comment:
```suggestion
    } else {
      Leanplum.countAggregator().incrementCount("operation_without_request_gzip");
    }
```

will let us monitor the rollout